test/images/scripts/lib/fedora.install

```diff
@@ -60,7 +60,9 @@ if [ -n "$do_install" ]; then
     packages=$(find build-results -name "*.rpm" -not -name "*.src.rpm" | grep -vF "$skip")
     rpm -U --force $packages
 
-    firewall-cmd --add-service=cockpit --permanent
+    if which firewall-cmd > /dev/null 2> /dev/null; then
```

Comment:
In [other places](https://github.com/cockpit-project/cockpit/blob/01e787eb37e19c86b7c390a8ba8813d5c5233fca/test/images/scripts/fedora-24.setup#L62) we use `type`. I think it would be good to be consistent.